package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@davesnx/styled-ppx",
-  "version": "0.54.1",
+  "version": "0.56.0",
```

Comment:
@davesnx This changed on its own without me actually editing it, is it OK?

Reply:
weird, but it's fine